Composer/packages/lib/shared/src/lgUtils/models/LgTemplateRef.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+import parseLgTemplateRef from '../parsers/parseLgTemplateRef';
+
+import { LgTemplateName, LgTemplateRefString, LgText } from './stringTypes';
+
+export default class LgTemplateRef {
+  name: LgTemplateName;
+
+  parameters: string[] | undefined;
+
+  constructor(name: LgTemplateName, parameters?: string[]) {
+    this.name = name;
+    this.parameters = parameters;
+  }
+
+  static parse(input: LgTemplateRefString): LgTemplateRef | null {
+    return parseLgTemplateRef(input);
+  }
+
+  toString(): LgTemplateRefString {
+    const paramsSuffix = Array.isArray(this.parameters) ? `(${this.parameters.join(',')})` : '';
+    return `[${this.name}${paramsSuffix}]`;
+  }
+
+  toLgText(): LgText {
```

Comment:
lgText, by the definition, don't contains "-", and i don't see any other places using this toLgText()? 

can we only keep one toString()?